dockerCompose/dockerComposeCompletionItemProvider.ts

```diff
@@ -55,12 +59,14 @@ export class DockerComposeCompletionItemProvider implements CompletionItemProvid
         return Promise.resolve([]);
     }
 
-    private suggestKeys(word: string): CompletionItem[] {
-        return Object.keys(DOCKER_COMPOSE_KEY_INFO).map(ruleName => {
+    private suggestKeys(word: string, useV2Schema: boolean = false): CompletionItem[] {
```

Comment:
We could definitely do that, but I generally think the way that auto-completion is implemented isn't the best long-term solution, so I just went with a simple fix for getting v2 support added. I filed #36 to track adding support for Compose v2.1 files, which would definitely necessitate tweaking how this works, and would be pretty trivial to do once needed.